README.md

```diff
@@ -27,6 +27,8 @@ Channel controller sync and promote resources from the target source to a channe
 
 ## Quick Start
 
+Please keep in mind, if you were using this operator before, you might need to migrate your resource api-group and version from `app.ibm.com/v1alpha1` to `multicloud-apps.io/v1`. Example can be found under the `examples` folder.
```

Comment:
Let's just write:

Keep in mind......

Also, does this mean all the APIs are changing?

Reply:
Yes, all the repo belongs to app-lifecycle is changing from `app.ibm.com/v1alpha1` to `multicloud-apps.io/v1`